client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -5,21 +5,42 @@ import useStyles from './dashboardStyles';
 import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
 import { useHistory } from 'react-router-dom';
-import NavBar from '../../components/NavBar/NavBar';
+import { useState } from 'react';
 import { useEffect } from 'react';
+import NavBar from '../../components/NavBar/NavBar';
+import getUserBoards from '../../helpers/APICalls/getUserBoards';
+import { Board } from '../../interface/Board';
+import OptionsDrawer from '../../components/OptionsDrawer/OptionsDrawer';
 
 export default function Dashboard(): JSX.Element {
   const classes = useStyles();
-
   const { loggedInUser } = useAuth();
   const { initSocket } = useSocket();
-
+  const [openDrawer, setOpenDrawer] = useState<boolean>(false);
+  const [boards, setBoards] = useState<Array<Board>>([]);
   const history = useHistory();
 
+  const getAllUserBoards = () => {
+    getUserBoards()
+      .then((data) => {
+        setBoards(data.boards);
+      })
+      .catch(() => setBoards([]));
+  };
+
+  useEffect(() => {
+    getAllUserBoards();
```

Comment:
since `getAllUserBoards` is not in the dependancy array, is this giving you a warning?